### DISCUSSION TITLE
cronjob /start and /finish

### DISCUSSION THREAD

**Discussion by wh4u** [healthchecks/healthchecks#942] [General]
Hello everyone.
I 've been experimenting with the selfhosted version of healthchecks. Great software !
Regarding Measuring Script Run Time, i can append the /start  to signal start as docs instructs.
How to syntax the cronjob to also signal the success finish?

**Comment by cuu508** [comment]
To signal success, remove `/start` from the URL, use no suffix. 

https://healthchecks.io/docs/http_api/#success-uuid
